<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profileWindow</class>
 <widget class="QWidget" name="profileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>318</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>318</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: none;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="profileCont_2">
     <property name="styleSheet">
      <string notr="true">#profileCont_2 { 
	background-image: url(:/imgs/welcomePic.png);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignRight">
       <widget class="QGroupBox" name="horizontalGroupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border : none ;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>16</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="pushButton_5">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>11</width>
             <height>11</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background : red; 
border-radius : 5px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="profileCont">
        <property name="styleSheet">
         <string notr="true">#profileCont { 
border:none; 

}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>21</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="ProfileImageLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>157</width>
             <height>161</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/imgs/Profile (2).png);
background : transparent </string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="ProfileUploadBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>228</width>
             <height>48</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">#ProfileUploadBtn{
	border:2px dotted #3663fd;
	border-radius: 5px;
	font-size: 15px;
	color: white;
	font-weight: bold;
    transition: 2s all
}
#ProfileUploadBtn:hover{
background-color : rgb(28, 129, 253); 
}
#ProfileUploadBtn:pressed{
border : 1px solid blue ; 

}</string>
           </property>
           <property name="text">
            <string>Upload profile picture</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="verticalGroupBox">
           <property name="styleSheet">
            <string notr="true">border:none;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="horizontalGroupBox">
              <property name="styleSheet">
               <string notr="true">border:none;</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="ProfileNameLabel">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#ProfileNameLabel{
	font-size: 15px;
	font-weight: bolder;
	background : transparent ;
	color : white ; 
	font-weight : bold  ;
}</string>
                 </property>
                 <property name="text">
                  <string>Your Profile Name</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="styleSheet">
               <string notr="true">color:white;
font-size: 14px;
border-radius:5px;
background-color:#21282c;
padding:5px;
font-weight:bold;
</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ProfileAboutLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#ProfileAboutLabel{
	font-size: 20px;
	font-weight: bolder;
	background : transparent;
	color : white ; 
	font-weight : bold; 
}</string>
           </property>
           <property name="text">
            <string>About</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="styleSheet">
            <string notr="true">color:white;
font-size: 14px;
border-radius:5px;
background-color:#21282c;
padding:5px;
font-weight:bold;
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="styleSheet">
            <string notr="true">#pushButton{background-color :#3663fd;
font-weight : bold ;
border-radius : 5px;
font-size : 15px; 
color:white;
padding: 5px;}
#pushButton:pressed {
    border: 1px solid white
}
</string>
           </property>
           <property name="text">
            <string>Save changes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="styleSheet">
            <string notr="true">#pushButton_2{background-color :rgb(203, 33, 36);
font-weight : bold ;
border-radius : 5px;
font-size : 15px; 
color:white;
padding: 5px;}
#pushButton_2:pressed {
    border: 1px solid white
}
</string>
           </property>
           <property name="text">
            <string>Logout</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
